getting-started/README.md

```diff
@@ -0,0 +1,113 @@
+# Get started with ONNX Runtime Training
+
+Train a PyTorch transformer model with ONNX Runtime.
+
+This sample trains the model defined here: https://pytorch.org/tutorials/beginner/transformer_tutorial.html. This is a transformer model composed of PyTorch transformer components such as the PositionalEncoder, TransformerEncoder and word Embedding.
+
+Trained using the Wikitext-2 dataset (from `torchtext.dataset`), the language model assigns a probability for the likelihood of a given word (or a sequence of words) to follow a sequence of words.
+
+_Note: [ORTTrainer API](https://github.com/microsoft/onnxruntime/blob/orttraining_rc1/orttraining/orttraining/python/ort_trainer.py#L480) is experimental. A new version of the API is under active development, and this sample will be updated to that API soon._
+
+This sample uses the base docker image `mcr.microsoft.com/azureml/onnxruntime-training:0.1-rc2-openmpi4.0-cuda10.2-cudnn7.6-nccl2.7.6` to train the model. The image contains the ONNX Runtime package, OpenMP and NVIDIA GPU dependencies. The docker image can be used either standalone or in Azure Machine Learning.
+
+The purpose of this sample is to demonstrate the concepts and usage of ONNX Runtime Training, rather than to show its performance capabilities. For more complex samples that do demonstrate improved training performance at scale, please refer to the other samples in this repo:
+
+* [NVIDIA BERT pre-training](../nvidia-bert)
+* [HuggingFace GPT-2 fine-tuning](../huggingface-gpt2)
+
+All instructions in this sample assume you have cloned this repo and are located in the root directory of the getting-started sample.
+
+```bash
+git clone https://github.com/microsoft/onnxruntime-training-examples
+cd getting-started
+```
+
+## Code walk through
+
+The training code is based on the original PyTorch training code, which is provided for reference in the `train.py` file.
+
+### Model
+
+The PyTorch model is defined in `model.py`.
+
+### Trainer class
+
+See `train_ort.py` for the ONNX Runtime Training code. In its `train_step` method, the ORTTrainer encapsulates one pass (forward pass, loss calculation, backward pass, and optimizer weight adjustment) of the training loop.
+
+Parameters and options for training are supplied to `ORTTrainer` when it is created:
+
+```python
+trainer = ORTTrainer(model,                       # model
+                     loss_with_flat_output,       # loss function
+                     model_description(),         # model description
+                     "SGDOptimizer",              # optimizer name
+                     None,                        # optimizer attributes
+                     learning_rate_description(), # learning rate description
+                     device,                      # device
+                     _opset_version=12)           # opset version
+```
```

Comment:
add partition optimizer flag - option for deepspeed_zero1.

Reply:
Added section on distributed training.